[2022-04-04 00:36:58,641] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.task_3 2022-04-04T00:36:52.254912+00:00 [queued]>
[2022-04-04 00:36:58,656] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.processing_tasks.task_3 2022-04-04T00:36:52.254912+00:00 [queued]>
[2022-04-04 00:36:58,656] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-04-04 00:36:58,657] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-04-04 00:36:58,657] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-04-04 00:36:58,689] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_3> on 2022-04-04T00:36:52.254912+00:00
[2022-04-04 00:36:58,691] {standard_task_runner.py:52} INFO - Started process 10489 to run task
[2022-04-04 00:36:58,703] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag.processing_tasks', 'task_3', '2022-04-04T00:36:52.254912+00:00', '--job-id', '102', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpgg_gh5l3', '--error-file', '/tmp/tmp39b2r3zb']
[2022-04-04 00:36:58,704] {standard_task_runner.py:77} INFO - Job 102: Subtask task_3
[2022-04-04 00:36:58,832] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.processing_tasks.task_3 2022-04-04T00:36:52.254912+00:00 [running]> on host airflowvm
[2022-04-04 00:36:59,007] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag.processing_tasks
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2022-04-04T00:36:52.254912+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-04T00:36:52.254912+00:00
[2022-04-04 00:36:59,010] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-04-04 00:36:59,010] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-04-04 00:36:59,028] {subprocess.py:75} INFO - Output:
[2022-04-04 00:37:02,036] {subprocess.py:83} INFO - Command exited with return code 0
[2022-04-04 00:37:02,135] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag.processing_tasks, task_id=task_3, execution_date=20220404T003652, start_date=20220404T003658, end_date=20220404T003702
[2022-04-04 00:37:02,208] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-04 00:37:02,228] {local_task_job.py:151} INFO - Task exited with return code 0
